wordpress-mobile/WordPress-Android
# Fastlane: Builds automation

This PR adds some lanes to handle the automatic builds of WPAndroid after the new versioning scheme.
This PR is related to https://github.com/wordpress-mobile/release-toolkit/pull/54.

### To test:
1. Run `bundle install`.
2. Run `bundle exec fastlane build_test_releases`.
3. At the prompt, verify that the versions are correct.
4. Verify that the right .aabs and .apks are built. Test them.
5. Run `bundle exec fastlane build_release`.
6. Verify that the lane exits with an error because the version name for the `vanilla` build is a beta one.
7. Checkout a new test branch out of the current one.
8. Change the version name for the `vanilla` build to a final one (remove the `-rc-1`).
9. Run `bundle exec fastlane build_release`.
10. Verify that the right .aab and .apk are built. Test them.

Note: Before merging, I'll update this PR with the final reference to `release-toolkit`.
 
Update release notes:
